app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb

```diff
@@ -9,7 +9,7 @@ def create_in_provider(manager_id, params)
 
       refresh(manager)
       find_by!(:manager_id => manager.id, :manager_ref => tower_object.id)
-    rescue AnsibleTowerClient::ClientError, ActiveRecord::RecordNotFound => error
+    rescue Exception => error
```

Comment:
@jameswnl http://daniel.fone.net.nz/blog/2013/05/28/why-you-should-never-rescue-exception-in-ruby/

Rescuing `Exception` will rescue a *LOT* of different kinds of errors.  If we're not going to classify the different types of errors, you probably want to rescue `StandardError` instead.